packages/@aws-cdk/aws-ecs-patterns/lib/base/application-load-balanced-service-base.ts

```diff
@@ -78,7 +78,8 @@ export interface ApplicationLoadBalancedServiceBaseProps {
    * The desired number of instantiations of the task definition to keep running on the service.
    * The minimum value is 1
    *
-   * @default 1
+   * @default - When creating the service, default is 1; when updating the service, default uses
+   * the current task number.
```

Comment:
Don't we still need to change https://github.com/aws/aws-cdk/blob/master/packages/%40aws-cdk/aws-ecs-patterns/lib/base/application-load-balanced-service-base.ts#LL371? (Same for the other base patterns -- looks like we're still setting "1" as the default, but this is now handled by CFN directly (https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ecs-service.html#cfn-ecs-service-desiredcount)

Reply:
As per @MrArnoldPalmer comment yesterday, we should avoid removing that property as it would be backwards breaking, however, we will no longer be using this.desiredcount to actually create the service, so it will just exist, and be removed as part of v2.